vibe_code_apps_13/scripts/run_linux_timing_gate.sh

```
@@ -128,20 +139,52 @@ python3 "$ROOT/scripts/cyclictest_summary.py" "$ART/cyclictest-loaded.txt" \
+    "## Assessments",
+    f"- measurement_execution: $MEASUREMENT_EXECUTION",
+    f"- stress_ng_execution: $STRESS_EXEC",
+    f"- haystack_before: $HAYSTACK_BEFORE",
+    f"- haystack_after: $HAYSTACK_AFTER",
+    f"- scheduling_threshold_assessment (idle): $SCHED_IDLE",
+    f"- scheduling_threshold_assessment (loaded): $SCHED_LOADED",
+    f"- scheduling_threshold_assessment (overall): $SCHED_OVERALL",
+    "- wire_timing_measured: false",
+    "- clause9_conformance: not_claimed",
+    "",
+    "Note: cyclictest `-m` means **mlockall**, not one worker per CPU.",
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)